user-guide/Basic_Functionality/Trending/Advanced_analytics_trending/Working_with_behavioral_anomaly_detection.md

```diff
@@ -8,6 +8,7 @@ uid: Working_with_behavioral_anomaly_detection
 >
 > - This feature requires a general database of type "Cassandra".
 > - Anomaly detection is only available for numeric parameters that are not part of [partial tables](xref:Table_parameters#partial-tables). It is also limited to at most 100,000 parameters per DMA.
+> - From DataMiner 10.3.3/10.4.0 onwards, anomaly detection is not available for discrete parameters. <!-- RN 35465 -->
```

Comment:
As anomaly detection for discrete parameters apparently doesn't make much sense (according to the task for this RN), maybe it would be better if we just changed this to "Anomaly detection is not available for discrete parameters." – ignoring that strictly speaking it was available in earlier versions, because it wasn't much use then anyway. Otherwise I think this note might bring up more questions than it answers.

Reply:
I agree with this. I would not want a user to try out anomaly detection on a discrete parameter as it will probably not give the expected results.